src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -1990,11 +1990,12 @@ JavaThreadStatus java_lang_VirtualThread::map_state_to_thread_status(int state)
       status = JavaThreadStatus::NEW;
       break;
     case STARTED :
-    case RUNNABLE :
     case RUNNING :
     case PARKING :
     case TIMED_PARKING:
+    case UNPARKED:
     case YIELDING :
+    case YIELDED:
```

Comment:
Is this new state `YIELDED` really needed?
It looks like `UNPARKED` is also appeared where `YIELDED` is used.